### ISSUE TITLE
[SF_ACC] Segmented Button Missing NVDA info

### ISSUE THREAD

**Issue by IrvinDelgado** [SAP/ui5-webcomponents#6402]
### **Bug Description**
When focusing on a selected Segmented Button, the information which says "Selected" is not announced. However, for the "Unselected" it works correctly.

### **Expected Behavior**
When focusing on a selected button "Selected" should be announce

### **Steps to Reproduce**
1. Go to [Segmented Button](https://sap.github.io/ui5-webcomponents/playground/components/SegmentedButton/)
2. Tab focus to selected button
3. Notice that the user does not know if Button is selected

### **Isolated Example**
If possible, please provide an isolated example by forking 

### **Context**
- UI5 Web Components version: 1.10.0
- OS/Platform: Windows/NVDA
- Browser: Chrome
- Affected component: Segmented Button

### **Log Output / Stack Trace / Screenshots**
<img width="767" alt="Screenshot 2023-02-01 at 4 30 51 PM" src="https://user-images.githubusercontent.com/62973847/216201800-cbfdd53d-da6b-4177-87ed-64077d6bd9da.png">
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-b,
can you take a look at this issue?

Best Regards,
Todor

**Comment by unazko** [comment]
Hello @IrvinDelgado,

We already do have a reported screen reader issue about the same behavior.
This is how a `listbox `with `option`'s work.

Fortunately the users will know, which are the selected options by not hearing "not selected".

Best regards,
Boyan
